### DISCUSSION TITLE
Select tags via htmx (like on Stackoverflow)

### DISCUSSION THREAD

**Discussion by guettli** [bigskysoftware/htmx#2145] [Q&A]
I like the UI of tags in Stackoverflow:

![image](https://github.com/bigskysoftware/htmx/assets/414336/55fd5cc1-849f-4d96-ad67-574d6c1e861e)

I checked the [examples](https://htmx.org/examples/) but could not find something like that.

Do you know how to get such UI via htmx?

**Comment by efosao** [comment]
If you're looking for the UI for these kinds of tags, it has nothing to do with HTMX. You will need to find or create HTML/JS/CSS or Web Component for this tag search behavior first, _then you can worry about how to get it working with HTMX_. 
HTMX is only concerned with the client/server fetch/replace behavior. 

### Some examples you can copy and adapt
- https://tailwindcomponents.com/component/multi-select
- https://tailwindcomponents.com/component/select-with-search
- Web Components - [Writing a tag input web component](https://blog.revillweb.com/writing-a-tag-input-web-component-fa7d028a8655)
- HTML/CSS/JQuery - https://codepen.io/reza__f/pen/povPEmd

When you have that completed, you can use HTMX to tie in and drive the tag search behavior, like I am doing over here in this HTMX demo I created.
- [bun.vauntly.com - HTMX Job Posts Demo](https://bun.vauntly.com/posts?tags=embedded,gitlab,junior)
